erizo/src/erizo/handlers/HandlerImporter.h

```diff
@@ -0,0 +1,41 @@
+//
+// Created by licode20 on 22/3/21.
+//
+
+#ifndef ERIZO_SRC_ERIZO_HANDLERS_HANDLERIMPORTER_H_
+#define ERIZO_SRC_ERIZO_HANDLERS_HANDLERIMPORTER_H_
+
+#include <string>
+#include <map>
+#include "../pipeline/Handler.h"
+#include "./logger.h"
+#include "handlers/LoggerHandler.h"
+
+
+
+namespace erizo {
+
+
+class HandlerImporterInterface {
+    DECLARE_LOGGER();
+ public:
+    void loadHandlers(std::vector<std::map<std::string, std::string>> customHandlers);
+    std::map<std::string, std::shared_ptr<erizo::CustomHandler>> handlers_pointer_dic = {};
+    std::vector<std::string> handler_order = {};
+};
+
+
+class HandlerImporter: public HandlerImporterInterface {
+    DECLARE_LOGGER();
+ public:
+    HandlerImporter();
+    void loadHandlers(std::vector<std::map<std::string, std::string>> customHandlers);
+ private:
+    enum HandlersEnum {LoggerHandlerEnum};
```

Comment:
To make this even easier for developers, this Enum could become a vector and have an initial step where the new Handlers should self-register to that vector. It could work if the HandlerImporter is a Singleton, for instance.

But we can do it in a different PR.